src/ro/sci/h9/Time.java

```diff
@@ -0,0 +1,77 @@
+package ro.sci.h9;
+
+/**
+ * This class represents a time (minutes and seconds).
+ *
+ * @author Emanuel Pruker
+ */
+public class Time implements Comparable<Time> {
+    private int seconds;
+
+    /**
+     * Constructs the time representing 0 seconds.
+     */
+    public Time() {
+        this(0);
+    }
+
+    /**
+     * Constructs the time from the number of seconds.
+     *
+     * @param seconds number of seconds
+     */
+    public Time(int seconds) {
+        this(0, seconds);
+    }
+
+    /**
+     * Constructs the time from the number of minutes and seconds.
+     *
+     * @param minutes number of minutes
+     * @param seconds number of seconds
+     */
+    public Time(int minutes, int seconds) {
+        this.seconds = seconds + 60 * minutes;
+    }
+
+    /**
+     * Construct a time from an input string.
+     *
+     * @param input the string representation of a time
+     * @return the time instance
+     */
+    public static Time fromString(String input) {
+        if (input == null) {
+            throw new IllegalArgumentException("It doesn't work with \"null\" ");
+        }
+        String[] parts = input.split(":");
+        if (parts.length != 2) {
+            throw new IllegalArgumentException("Invalid input!");
+        }
+
+        return new Time(Integer.parseInt(parts[0]), Integer.parseInt(parts[1]));
```

Comment:
what if time parts[0] is not an int?
What is parts[0]=12312321? What if parts[0]=123123?